<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchRow</class>
 <widget class="QWidget" name="MatchRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>25</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>25</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>25</height>
   </size>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="lObject">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="reghaabat.qrc">:/images/object.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lMatch">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="reghaabat.qrc">:/images/match.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lFine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #9d261d</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="reghaabat.qrc"/>
 </resources>
 <connections/>
</ui>
